### ISSUE TITLE
Input: adding icon to Input with action clears border radius on the action button

### ISSUE THREAD

**Issue by ilyastam** [Semantic-Org/Semantic-UI-React#2506]
**Steps to Reproduce**
use this page to reproduce: https://react.semantic-ui.com/elements/input#input-example-action

without icon:

![image](https://user-images.githubusercontent.com/869125/35949050-7671e20c-0c24-11e8-9665-6177d5352082.png)

with icon:
notice that button now doesn't have the border radius
![image](https://user-images.githubusercontent.com/869125/35949086-abcba94c-0c24-11e8-8e83-2f5b950b5312.png)

**Comment by brianespinosa** [comment]
@ilyastam confirmed that this is valid. For the CSS to work, the icon should be rendered in between the `<input>` and `<button>` as shown below:

```
<div class="ui action left icon input">
  <input type="text" placeholder="Search...">
  <i aria-hidden="true" class="search icon"></i>
  <button class="ui button" role="button">Search</button>
</div>
```
Which will give us this: 
![image](https://user-images.githubusercontent.com/1161781/36003578-891e2b9e-0ce3-11e8-9d84-145e9bde41e2.png)

The rounding of the corners comes from the :first-child/:last-child CSS selectors in the SUI styles.

**Comment by Intregrisist** [comment]
@brianespinosa Yeah, I have a PR for this can we get this merged #2507.
